.github/workflows/community-pr.yml

```diff
@@ -0,0 +1,42 @@
+name: Label community PRs
+on:
+  pull_request_target:
+    types: [opened]
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  add_milestone:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'dotnet/roslyn-analyzers' && github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER' }}
```

Comment:
📝 The V4 API (GraphQL) seems to be different from the V3 API for this field. The `author_assocation` field is shown in examples for the V3 API, but GitHub (in line with industry standards) has opted to leave the majority of API behaviors undocumented, so it's not clear what the possible values of this enumeration are or what their meaning is.

Reply:
Is verifying the repo matches necessary?  Presumably since this is an action for roslyn-analyzers it should only be triggered by PRs to roslyn analyzers?  Or can this be triggered by forks

Reply:
I found this which seems to validate the author_association checks too https://docs.github.com/en/graphql/reference/enums#commentauthorassociation

Reply:
Those are the values returned by the V4 API. If you use GraphQL to get pull request #5569, you get `MEMBER` for the association:
![image](https://user-images.githubusercontent.com/1408396/135542371-63eb8c8b-0d87-4df1-be59-9447d32ea64b.png)

However, if you use the V3 API to get the [same pull request](https://api.github.com/repos/dotnet/roslyn-analyzers/pulls/5569), it returns an `"author_association": "CONTRIBUTOR"` which has now broken this rule.